.github/workflows/linters.yml

```diff
@@ -0,0 +1,13 @@
+name: Linter Checks
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  license-headers-linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check license Headers
+        uses: apache/skywalking-eyes/header@v0.6.0
```

Comment:
Should we pin the commit of this 3rd party action? Looks like it's part of the apache foundation so it may be trustworthy enough.

Reply:
Yes seems trustworthy and widely used. Will leave it to Andrei's discretion about pinning the commit or not